cmd/contacts.go

```diff
@@ -26,11 +39,73 @@ func (x *contactsCmd) Name() string {
 }
 
 func (x *contactsCmd) Short() string {
-	return "Get, add, and list local contacts"
+	return "Manage contacts"
 }
 
 func (x *contactsCmd) Long() string {
-	return "Get, add, and list local contacts."
+	return `
+Use this command to add, list, get, and remove local contacts and find other contacts on the network.
+`
+}
+
+type addContactsCmd struct {
+	Client ClientOptions `group:"Client Options"`
+}
+
+func (x *addContactsCmd) Usage() string {
+	return `
+
+Add to known contacts.
+`
+}
+
+func (x *addContactsCmd) Execute(args []string) error {
+	setApi(x.Client)
+
+	fi, err := os.Stdin.Stat()
+	if err != nil {
+		return err
+	}
+	if (fi.Mode() & os.ModeCharDevice) != 0 {
+		return errMissingStdin
+	}
+
+	input, err := ioutil.ReadAll(os.Stdin)
```

Comment:
Because adding a contact is fairly complex in terms of there structure, this is limited to take input from stdin. So, you could pipe search results or a file.

Reply:
good call... will update w/ next PR